splash/qtrender_lua.py

```diff
@@ -80,7 +80,7 @@ def get_last(self):
 
 
 def command(table_argument=False, sets_callback=False,
-            decode_arguments=True):
+            decode_arguments=True, flag=False):
```

Comment:
could you please use a longer name for `flag` variable? e.g. `errors_as_flags` or something like this (I'm not exactly sure about its meaning)?